HISTORY.md

```diff
@@ -0,0 +1,43 @@
+# Cheerio releae notes
+
+## 1.0.0-rc.1 (2017-05-27)
+
+This release changes Cheerio's default parser to [the Parse5 HTML
+parser](https://github.com/inikulin/parse5). Parse5 is an excellent project
+that rigorously conforms to the HTML standard. It does not support XML, so
+Cheerio continues to use [htmlparser2](https://github.com/fb55/htmlparser2/)
+when working with XML documents.
+
+This switch addresses many long-standing bugs in Cheerio, but some users may
+experience slower behavior in performance-critical applications. In addition,
+htmlparser2 is more forgiving of invalid markup which can be useful when input
+sourced from a third party and cannot be corrected. For these reasons, this
+release offers users the ability to continue using htmlparser2 through explicit
+configuration.
+
+### Migrating from 0.x
+
+**Re-enabling htmlparser2** As noted above, the new Parse5 library may not be
```

Comment:
Actually we've changed API a bit according to https://github.com/cheeriojs/cheerio/pull/985#issuecomment-289225391

Officially there is only one option: `xml` that can be either boolean or an object that contains htmlparser2 settings. However, for back compatibility, I've left ability to provide options on the `options`. `useHtmlParser2` option is used internally for DOM cloning, and we decided to not make it a part of public, however I don't have any problems with that.